README.md

```diff
@@ -163,3 +163,13 @@ const { red } = createColors(options.enableColors)
 
 On `undefined` argument, `createColors` will use value
 from color support auto-detection.
+
+&nbsp;
+
+--------
+
+*nanocolors* are primarily distributed under the terms of the [MIT license]. See
```

Comment:
Do we really need to mention it in docs?

GitHub and npm already shows it in UI.

Reply:
Well.. it's your call. I'll revert it if you don't want.

Reply:
Let’s remove it.

I want to keep docs focused on end-user needs.